keras/layers/core.py

```diff
@@ -592,10 +592,21 @@ def antirectifier_output_shape(input_shape):
         model.add(Lambda(antirectifier,
                          output_shape=antirectifier_output_shape))
     ```
+    ```python
+        # add a layer that returns the hadamard product
+        # of the two input tensors
+
+        def hadamard_product(tensors):
+            return tensors[0]*tensors[1]
+
+        x1 = Dense(32, activation="relu", input_dim=3)(input_1)
```

Comment:
` x1 = Dense(32)(input_1)` is enough for an example.

Reply:
okay, will do.